### ISSUE TITLE
Diacritics onto Noto Sans Canadian Aboriginal, and additional positions for Finals 

### ISSUE THREAD

**Issue by CJLippert** [googlefonts/noto-fonts#1558]
NotoSansCanadianAboriginal-Regular_1.ttf
Version 1.03 uh
Installed: Thursday, ‎August ‎04, ‎2016, ‏‎11:03:34 AM onto a Win7 machine with Version	6.1.7601 Service Pack 1 Build 7601

**1a.** When using MS Word (of MSOffice Version 14.0.7232.5000 (32-bit)) [yes, I know my machine is getting to be old], the combining diacritics found in the NOTO Font family does not combine with the UCAS glyphs.  This is an issue because although many glyphs are precomposed with either the dot above or ring above, having the ability to place the combining diacritics onto other characters that do not have the precomposed forms would be advantageous, especially for the various Athapaskan languages who then could use the combining acute, combining grave, combining circumflex, combining breve, combining diaeresis, and combing ring above to notate their pitch accent, provide differentiation to a character that may map out to two or more sound values in speech, as well as providing potential cross-language use of certain glyph form sets. This would also allow for the Dot Above to be placed on the ᐁ ᐯ ᑌ ᑫ ᒉ ᒣ ᓀ ᓓ ᓭ ᔐ ᔦ and on barred ᓭ (see below) for Syllabics use in Fox and Sauk.

**1b.** Create in the Private Use Area the following combining diacritical marks specific to UCAS: 
  Combining Middle Dot (found already in precomposed shapes such as ᗎ, ᘌ, ᘲ, and ᙫ)
  Combining Small Vertical Line (found already in precomposed shapes such as ᘅ, ᘑ, ᗸ, ᗫ)
  Combining Ojibwe L (a ᓫ which as a diacritic that sits above ᓀ ᓂ ᓄ ᓇ)
  Combining Ojibwe R (a ᕑ which as a diacritic that sits above ᓀ ᓂ ᓄ ᓇ)

**2.** Unlike the full-sized glyphs that typically represents an open syllable, the Finals that represent syllable coda typically are placed in a superscript position. However, in Athapascan use, the Finals need the ability to placed at the superscript position, mid-line position, and low line position, where some identically shaped Finals end up with different sound values depending on their placement height.  I would like to suggest NOTO shift these Finals to a mid-line position, which then could be easily superscripted or subscripted for these contextual variations.   Currently, only these two are available as a mid-line Finals: ᕀ ᕁ but some systems will display that identically to these: ᐩ ᕽ.  Other finals need the ability to be placed at different heights.

**3.** Some of the current glyphs by design are asymmetrical or are missing forms.  I would like to suggest filling in these gaps in the Private Use Areas, such that the missing forms that result in what otherwise would be an asymmetrical representation of a shape set would then, at least from a set standpoint, would have the necessary pieces to make each set be symmetrical. Some forms that would be included in this void-filling would include 
  The full-height form-equivalent to ᔿ to go with ᓫ ᓬ and ᕑ ᕒ, and their W-shaped counterpart to complete the set.
 Complete the ᖠ ᖢ ᖤ ᖦ set to be in parallel with ᓓ ᓕ ᓗ ᓚ ᓪ set.
 Barred ᓭ ᓯ ᓱ ᓴ ᔅ set to be in parallel with ᖧ ᖨ ᖪ ᖬ ᖮ set.
 Barbed ᓭ ᓯ ᓱ ᓴ ᔅ set to be in parallel with ᖠ ᖢ ᖤ ᖦ set.
 Reversed ᘖ to complete ᘔ ᘕ ᘖ and inverted ᘚ to complete ᘗ ᘚ ᘛ
 Alternate ᖕ based on the ᓐ ᒃ ligature rather than on the ᓐ ᒡ ligature
 Alternate ᖖ based on the doubling of the ᓐ ᒃ ligature
 A ᖖᒉ ligature to complete the ᙱ ᙳ ᙵ set
 Superscripted ᐊ and ᐅ to complete the ᐞ as a Final.

**4.** Currently missing from Unicode and would be nice to have also in the Private Use Area is the now obsolete SP series (which resembles Z {reversed Z} N И in form).
